spec/Vehicle/Vehicle.vspec

```diff
@@ -385,7 +385,8 @@ CurrentLocation.Heading:
   min: 0
   max: 360
   unit: degrees
-  description: Current magnetic compass heading.
+  description: Current heading relative to geographic north.
+               0 = North, 90 = East, 180 = South, 270 = West.
 
 CurrentLocation.Accuracy:
```

Comment:
```suggestion
CurrentLocation.HorizontalAccuracy:
```
To have the pair {horizontal, vertical} accuracy

Reply:
In the description the term "level" could be removed, as this might be misinterpreted to be a enum like data type.